src/main/resources/templates/admin/category/add.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head th:replace="admin/template :: head(title = 'EduQuiz - Admin')">
```

Comment:
Use the complete syntax of fragment expressions
```suggestion
<head th:replace="~{admin/template :: head(title = 'EduQuiz - Admin')}">
```